app-pda/libusbmuxd/libusbmuxd-1.0.10.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A client library to multiplex connections from and to iOS devices"
+DESCRIPTION="libusbmuxd is a client library to multiplex connections from and \
+to iOS devices by connecting to a socket provided by a usbmuxd daemon"
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2013-2014 Nikias Bassen
+	2009-2014 Martin Szulecki"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libusbmuxd-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1aa21391265d2284ac3ccb7cf278126d10d354878589905b35e8102104fec9f2"
+PATCHES="libusbmuxd-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libusbmuxd$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libusbmuxd$secondaryArchSuffix = 4.0.0 compat >= 4
+	cmd:iproxy$secondaryArchSuffix = 4.0.0 compat >= 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libplist$secondaryArchSuffix
+	lib:libplist++$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libusbmuxd${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libusbmuxd$secondaryArchSuffix = 4.0.0 compat >= 4
+	"
+REQUIRES_devel="
+	libxml2$secondaryArchSuffix
+	libplist$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libxml2$secondaryArchSuffix
+	devel:libplist$secondaryArchSuffix
+	devel:libplist++$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake_1.13
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sh
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	aclocal-1.13 -I m4
+	autoconf
+	automake-1.13
```

Comment:
Please remove these lines:
```
	libtoolize --force --copy --install
	aclocal-1.13 -I m4
	autoconf
	automake-1.13
```
(This is OK because we no longer need to patch Makefile.am.)